### ISSUE TITLE
Why different total parameter numbers when comparing those baselines

### ISSUE THREAD

**Issue by santisy** [lif314/X-KANeRF#1]
Hello,

Thanks for your contribution. Great work!
I am just wondering that why in the _Performance Comparison_ section, the layer params are not approximately the same? At least we should increase the basic MLP NeRF's parameter to approximately the same as others?

Best

**Comment by lif314** [comment]
> Hello,
> 
> Thanks for your contribution. Great work! I am just wondering that why in the _Performance Comparison_ section, the layer params are not approximately the same? At least we should increase the basic MLP NeRF's parameter to approximately the same as others?
> 
> Best

Thank you for your great advice! I'm also conducting more tests while ensuring the correctness of the code. However, we can still draw some useful conclusions from the current comparative method (with the same layers). For instance, although MLPs are faster, their parameter utilization is lower. With the same parameters, they require more layers, which in larger models may lead to issues such as vanishing or exploding gradients, long-term dependencies, etc.

**Comment by lif314** [comment]
> Hello,
> 
> Thanks for your contribution. Great work! I am just wondering that why in the _Performance Comparison_ section, the layer params are not approximately the same? At least we should increase the basic MLP NeRF's parameter to approximately the same as others?
> 
> Best

Thanks again! I have added the `Nerfacto-MLP-Big` eval result.  To achieve the 'same' parameters as KAN, we can either add more layers or increase the hidden dimensions. I've tested only adding more layers, but the experimental results aren't promising. In the experiments, increasing hidden dimensions, like (8-->64), seems to be a better solution. However, adding more layers could better highlight KAN's parameter efficiency. (Assuming, of course, that there are no bugs in my code!)
